crates/kilt-dip-support/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2023 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+// TODO: Crate documentation
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub mod did;
+pub mod merkle;
+pub mod state_proofs;
+pub mod traits;
+pub mod utils;
+
+mod export;
```

Comment:
Why is it not a good reason to have a module? We can make the module private and the elements inside it public, and export them. I might agree on the naming (better suggestions?), but I don't think it's a terrible idea. Maybe I'm biased by how Parity does it? E.g., XCM has a `latest` module where they re-export the latest XCM definitions.

Reply:
https://en.wikipedia.org/wiki/Cohesion_(computer_science)#Types_of_cohesion

Grouping by what is exported would be "Coincidental cohesion" IMO.
Maybe this could be called verifier? Having a module named `export` is fine. But only put the exports in their. Not the actual code.

Reply:
Good suggestion. I renamed it to `verifier`, and also made the module public, while re-exporting its components. https://github.com/KILTprotocol/kilt-node/pull/494/commits/c79ea0f36cbb9ad787db6ab8b0d7b747acde1884